applications/nrf5340_audio/src/bluetooth/Kconfig

```diff
@@ -158,14 +158,88 @@ config BLE_ISO_TEST_PATTERN
-	 data will be an incrementing value ranging from 0-255 and repeating.
-	 Note that enabling this feature will disable the audio stream.
+	  This will transmit the same amount of data as an audio stream, but the
+	  data will be an incrementing value ranging from 0-255 and repeating.
+	  Note that enabling this feature will disable the audio stream.
 
 config BLE_ISO_RX_STATS_S
 	int "Interval in seconds to print BLE ISO RX stats. 0 to deactivate"
 	default 0
 
+choice BT_AUDIO_UNICAST_BAP_CONFIGURATION
+	prompt "Unicast codec configuration"
+	depends on TRANSPORT_CIS
+	default BT_AUDIO_UNICAST_RECOMMENDED
+	help
+	  Select the unicast codec configuration as given in
+	  Table 5.2 of BAP Specification, USB only supports BT_AUDIO_UNICAST_RECOMMENDED
+
+config BT_AUDIO_UNICAST_RECOMMENDED
+	bool "Recommended unicast codec capability"
+	depends on TRANSPORT_CIS
+	help
+	  Recommended unicast settings for the nRF5340_audio application
+	  48kHz, 96kbps, 2 retransmits, 20ms transport latency, and 10ms presentation delay
+
+config BT_AUDIO_UNICAST_16_2_1
+	bool "16_2_1"
+	depends on TRANSPORT_CIS
+	help
+	  Unicast mandatory codec capability 16_2_1
+	  16kHz, 32kbps, 2 retransmits, 10ms transport latency, and 40ms presentation delay
+
+config BT_AUDIO_UNICAST_24_2_1
+	bool "24_2_1"
+	depends on TRANSPORT_CIS
+	help
+	  Unicast mandatory codec capability 24_2_1
+	  24kHz, 48kbps, 2 retransmits, 10ms transport latency, and 40ms presentation delay
+endchoice
+
+choice BT_AUDIO_BROADCAST_BAP_CONFIGURATION
+	prompt "Broadcast codec configuration"
+	depends on TRANSPORT_BIS
+	default BT_AUDIO_BROADCAST_RECOMMENDED
+	help
+	  Select the broadcast codec configuration as given
+	  in Table 6.4 of the BAP specification, USB only supports BT_AUDIO_BROADCAST_RECOMMENDED
+
+config BT_AUDIO_BROADCAST_RECOMMENDED
+	bool "Recommended Broadcast codec capability"
+	depends on TRANSPORT_BIS
+	help
+	  Recommended broadcast settings for the nRF5340_audio application
+	  48kHz, 96kbps, 4 retransmits, 20ms transport latency, and 10ms presentation delay
+
+config BT_AUDIO_BROADCAST_16_2_1
+	bool "16_2_1"
+	depends on TRANSPORT_BIS
+	help
+	  Broadcast mandatory codec capability 16_2_1
+	  16kHz, 32kbps, 2 retransmits, 10ms transport latency, and 40ms presentation delay
+
+config BT_AUDIO_BROADCAST_24_2_1
+	bool "24_2_1"
+	depends on TRANSPORT_BIS
+	help
+	  Broadcast mandatory codec capability 24_2_1
+	  24kHz, 48kbps, 2 retransmits, 10ms transport latency, and 40ms presentation delay
+
+config BT_AUDIO_BROADCAST_16_2_2
+	bool "16_2_2"
+	depends on TRANSPORT_BIS
+	help
+	  Broadcast mandatory codec capability 16_2_2
+	  16kHz, 32kbps, 4 retransmits, 60ms transport latency, and 40ms presentation delay
+
+config BT_AUDIO_BROADCAST_24_2_2
+	bool "24_2_2"
+	depends on TRANSPORT_BIS
+	help
+	  Broadcast mandatory codec capability 24_2_2
+	  24kHz, 48kbps, 4 retransmits, 60ms transport latency, and 40ms presentation delay
```

Comment:
Very nicely structured. I see that the xx_x_x format is from the spec so that is fine. Though, I still find it challenging to see exactly how the last _2_2 maps to the concrete settings.

Reply:
That is why I wrote the actual settings below.

Reply:
I see, then I suggest this format: 24: 24 kHz, _2: <...> _2: <...>

Reply:
I disagree, as the 2 has nothing to do with the actual setting, it is merely the index of the preset. By writing '_2: xxxx' users might think this is always this setting, which it's not.